data/campaigns/Son_Of_The_Black_Eye/scenarios/05_To_the_Harbor_of_Tirigaz.cfg

```diff
@@ -237,7 +237,7 @@
 
         [message]
             speaker="Kapou'e"
-            message= _ "Grüü is right. It is not wise to let these undead remain here cutting off our supply lines and path of retreat. In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats in our rear."
+            message= _ "Grüü is right. It is not wise to let these undead remain here, cutting off our supply lines and path of retreat. In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats in our rear."
```

Comment:
Given the fuss over small changes in text and the corresponding overhead for translators, I'll leave this alone for now but I just noticed that 'best not to leave any threats in our rear' should perhaps be reworded to '**to** our rear'. The former sounds like it could be mistaken for a reference to the buttocks...

Reply:
The original text:
> In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats in our rear.

Using "to our rear":
> In these days of turmoil, we don’t know what lies ahead, so it is best not to leave any threats to our rear.

Using "behind us":
> In these days of turmoil, we don’t know what lies **ahead**, so it is best not to leave any threats **behind** us.

What do you think about the sound of using "behind". It blends well as the antonym of the previously used word "ahead", and my feeling is that it is closer in meaning to the original text.

Reply:
I can see the symmetry in ahead/behind, but I think the original wording works better for me. Regardless, my point is that small changes like this are going to make some translators unhappy so I suppose it's better off left unchanged until and unless it is included in a bunch of other changes (or perhaps I learn how to use pofix properly). I've started keeping notes on potential changes like this.

(At any rate we're in the middle of a string freeze for 1.16.)